Add Parachains > Get Started page (#1042)
polkadot-developers/polkadot-docs: parachains/index.md

Reviewer: This page is a bit overwhelming. We're throwing a ton of links at people. It basically is just replicating what the user sees on the left navigation. I think we need to take a step back and think about what we really want on this page because as it is now it gives the user a bunch of different directions to get started, it doesn't actually help them achieve anything to get started... if that makes sense

Author: I am open to whatever but that is what **they** wanted right:

<img width="335" height="81" alt="Screenshot 2025-10-07 at 10 34 22 AM" src="https://github.com/user-attachments/assets/b2167903-220d-4b42-856f-aaa4adcc2785" />

A "cookbook-style" index.

---

> title: Get Started

Reviewer: Let's add some more context here for SEO purposes, like what are we getting started with?

Author: Added.

> description: Practical examples and tutorials for building and deploying parachains on Polkadot, from launching to customization and cross-chain messaging.

Reviewer: ```suggestion
description: Practical examples and tutorials for building and deploying Polkadot parachains, covering everything from launch to customization and cross-chain messaging.
```

I think "launching to customization" sounded a little off. I don't know anyways, my AI friend generated something similar 🤷‍♀️

categories: Basics, Parachains
---

# Get Started

> This resource provides examples and tutorials for building parachains on Polkadot. Each tutorial focuses on specific aspects of parachain development while providing hands-on examples and practical usage patterns.

Reviewer: I'm struggling with this tutorial terminology thing that keeps popping up. @dawnkelly09 did a learning session(a while ago at this point) on diataxis and the difference between different types of content, which I think is important to know and helpful for the way we think about classifications now.

The difference between tutorial and guides are subtle, but important.

**Tutorials**: Guided, hands-on lessons designed to help someone achieve something concrete. More learning-oriented — they’re like a “first experience” that builds confidence.
**Guides (How-to guides)**: Practical step-by-step instructions to help someone accomplish a specific task. They’re problem-oriented, not primarily about teaching.

I think there needs to be some discussion about the plan because there's a major disconnect in my mind about it. Maybe we can hijack the end of the sprint meeting, or stay later a few minutes to discuss this.

## Quick Start Tutorials

Quick start tutorials help developers set up and interact with the Polkadot parachain ecosystem using various tools and frameworks.

> | Tutorial | Tools | Description |

Reviewer 2: Overall, it looks good. What do you think about centering the content of the tables? I believe this can be done using the following syntax:

```
| Tutorial | Tools | Description |
|:-----:|:-----:|:-----------:|
```

| [Unlock Parachains](/parachains/runtime-maintenance/unlock-parachains){target=\_blank} | Understand parachain lifecycle and unlock mechanisms |

## Interoperability

Configure your parachain for cross-chain communication using XCM (Cross-Consensus Messaging).

| Tutorial | Description |
|-------|-------------|
| [Open HRMP Channels Between Parachains](/parachains/interoperability/channels-between-parachains){target=\_blank} | Establish communication channels with other parachains |
| [Open HRMP Channels with System Parachains](/parachains/interoperability/channels-with-system-parachains){target=\_blank} | Connect with Asset Hub and other system parachains |

## Integrations

Integrate your parachain with essential ecosystem tools and services.

| Tutorial | Description |
|-------|-------------|
| [Wallets](/parachains/integrations/wallets){target=\_blank} | Integrate wallet support for user interactions |
| [Indexers](/parachains/integrations/indexers){target=\_blank} | Set up indexing solutions for querying blockchain data |
| [Oracles](/parachains/integrations/oracles){target=\_blank} | Connect your parachain to off-chain data sources |

> ---

Reviewer 2: I'm not a big fan of adding this, since we haven't been using it throughout the docs. But I'd defer this to the regulators.

## Additional Resources

> - [Polkadot SDK Documentation](https://paritytech.github.io/polkadot-sdk/master/polkadot_sdk_docs/polkadot_sdk/index.html)
> - [Polkadot Wiki - Parachains](https://wiki.polkadot.network/docs/learn-parachains)

Reviewer 2: ```suggestion
- [Polkadot SDK Documentation](https://paritytech.github.io/polkadot-sdk/master/polkadot_sdk_docs/polkadot_sdk/index.html){target=\_blank}
- [Polkadot Wiki - Parachains](https://wiki.polkadot.network/docs/learn-parachains){target=\_blank}
```

> ## Community Support

Reviewer: I don't think this is relevant for this page

Author: Removed.

Need help? Connect with the parachain developer community:

> - [Polkadot Stack Exchange](https://polkadot.stackexchange.com/)
> - [Polkadot Discord](https://dot.li/discord)

Reviewer 2: ```suggestion
- [Polkadot Stack Exchange](https://polkadot.stackexchange.com/){target=\_blank}
- [Polkadot Discord](https://dot.li/discord){target=\_blank}
```
